src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/wiremock/SentencingAdjustmentsApiMockServer.kt

```diff
@@ -44,6 +45,273 @@ class SentencingAdjustmentsApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
+  fun stubAdjustmentsGet() {
+    stubFor(
+      get(urlPathMatching("/adjustments")).willReturn(
+        aResponse()
+          .withHeader("Content-Type", "application/json")
+          .withBody(
+            // language=json
+            """
+            [
```

Comment:
just cut and paste from Postman results for a Dev result, do something better once we need to start writing some tests